src/components/AppHeader/index.scss

```diff
@@ -15,13 +15,21 @@
     &__login-btn {
         color: white;
         background-color: #3C6BFF;
-        width: 174px;
-        height: 100%;
-        border-radius: 25px;
     }
 
     &__actions {
         display: flex;
         gap: 40px;
     }
+
+    &__button-action {
```

Comment:
I see that you already styled the button here https://github.com/Irina-Suprun/new-coffee/pull/3/files#diff-6d1ac1e7072d002fd68fa47aec09fccdbdd03bfc6b3366e3a7bd5fd96239abf3R69

It seems that you only need a restyled size, don't you?

Reply:
I thought that if I don't use all rule I need to create new

Reply:
I`m editing it